src/specializations/specializations.service.ts

```diff
@@ -14,4 +14,9 @@ export class SpecializationsService {
   create(createSpecializationInput: CreateSpecializationInput) {
     return this.specializationRepository.save(createSpecializationInput);
   }
+
+  async findOne(id: string) {
```

Comment:
please provide return types, also you can just `return this.specializationRepository.findOne(id);`
lets check your linter config